test/unit/lib/plugins/aws/package/compile/events/alb/index.test.js

```diff
@@ -116,4 +116,80 @@ describe('lib/plugins/aws/package/compile/eents/alb/index.test.js', () => {
       );
     });
   });
+
+  describe('path should be optional', async () => {
+    const baseEventConfig = {
+      listenerArn:
+        'arn:aws:elasticloadbalancing:' +
+        'us-east-1:123456789012:listener/app/my-load-balancer/' +
+        '50dc6c495c0c9188/f2f7dc8efc522ab2',
+    };
+
+    const {
+      cfTemplate: { Resources: myCfResources },
+    } = await runServerless({
```

Comment:
Let's not configure new `runServerless` run (they're expensive) but add function cases to already preconfigured one above

Reply:
Any suggestions on the negative test? I added a `runServerless` block to catch the error when validation fails

Reply:
Validation error indeed need to be tested with individual `runServerless` run, then best is to then ensure `code` on error, and test it that way: https://github.com/serverless/serverless/blob/44d398694fc803bb45a07e7bcc0d6cd9e572a704/test/unit/lib/plugins/aws/package/compile/events/httpApi/index.test.js#L437-L465